go.mod

```diff
@@ -40,11 +40,11 @@ require (
 	github.com/hashicorp/vault/api v1.0.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/influxdata/cron v0.0.0-20201006132531-4bb0a200dcbe
-	github.com/influxdata/flux v0.123.0
+	github.com/influxdata/flux v0.123.1-0.20210729174141-a0db406d1fae
```

Comment:
Are we allowed to update OSS to a specific hash without performing a release? My understanding was that we don't do this.

Reply:
I just asked the edge team and they're fine with it. They'll make sure that the version is a released version before they perform a release but it's fine to use revisions for things like breaking changes.